main.css

```diff
@@ -4,6 +4,36 @@
     box-sizing: border-box;
 }
 
+/* Mobile Styles */
+@media only screen and (max-width: 400px) {
+    body {
+        font-family: Arial;
+        color: black;
+        background-color:#FFCB77;
+        font-weight: normal;
+        font-size: 14px;
+    }
+  }
+  
+  /* Tablet Styles */
+  @media only screen and (min-width: 401px) and (max-width: 960px) {
+    body {
+      background-color: #F5CF8E; /* Yellow */
+    }
+  }
+  
+  /* Desktop Styles */
+  @media only screen and (min-width: 961px) {
+    body {
+        font-family: Arial;
```

Comment:
🔧 The font-family should probably not be declared within the media query. If you add it as a declaration on line 6, you won't have to repeat yourself.

Reply:
I will go in and try to make some corrections @nellarro @dontbecypher16

Reply:
@nellarro I finally sync the repo but I only see the current changes that includes the footer. How do I see and make changes to Richard's @dontbecypher16 file if its not merged?

Reply:
@thetechtakeover @dontbecypher16 @JLevelston @nojocode Hey should we jump on a quick call regarding this project since it's Saturday evening. I've also set up media queries for phone, tablet, and desktop because my issue needs them

Reply:
sure thats no problem @codingpixi I actually just submitted my pull request for the background color that included media query for mobile, tablet and desktop. We can choose one and just go with it so we dont have different sizes. @dontbecypher16

![Media Query](https://user-images.githubusercontent.com/47233500/91648930-dc681480-ea3b-11ea-8165-0d240a128252.PNG)

        
```

Reply:
thats what I went based off suggestions from W3School sizes @ codingpixi